packages/rum-core/src/common/service-factory.js

```diff
@@ -52,20 +49,12 @@ class ServiceFactory {
         serviceFactory.getService('LoggingService')
       )
     })
-
-    this.registerServiceInstance('PatchUtils', patchUtils)
```

Comment:
was never used, Let's add only if its required. We expose most of the modules from @elastic/apm-rum-core, So if we need those, We can directly . import and use . it.